docs/api-testing/project-access.md

```diff
@@ -44,31 +44,12 @@ You create projects and define project settings on the Projects page, and edit p
 1. Optionally, select one or more **Tags**, or click **Add custom tag** to add a new tag.
 1. Optionally, enter a **Description** of the project.
 1. Optionally, enter **Notes** about the project.
-1. Select the **Access** level for the project. If you are a team member you can select your team or organization-wide access. If you are an organization admin, you can select any team or organization-wide access. This selection cannot be changed once the project is saved.
+1. Select the **Access** level for the project. If you are a team member you can select your team or organization-wide access. If you are an organization admin, you can select any team or organization-wide access.
 1. Click **Save**.
 
 ### Manage Access to Projects
 
-Once a project has been created and saved, the access level (team or organization-wide access) is uneditable.
-
-If an organization admin deletes a team from Sauce Labs [Accounts > Team Management](/basics/acct-team-mgmt/managing-user-info/) before managing project access in API Testing, the access level defaults to Organization because the access setting is required. However, the project will still be available only to the original team and does not change to organization-wide access.
-
-If you need to change access to a project, you have the following options.
-
-#### Update Teams in Sauce Labs Accounts > Team Management
-
-If access is assigned to a team, you can add or remove team members from the team, which is outside of APIT in the Account area in Sauce Labs (go to Accounts > Team Management). You must be an organization admin to work with teams in Team Management.
-
-#### Update Project Access in API Testing
-
-You can create a new project with the access you want, and then delete or rename the original project as follows. You must have team access to the team you want to configure, or be an organization admin to do this.
-
-1.  Create a new project with the access (team or organization) you want.
-2.  Export the tests from the original project and import them to the new project.
-3.  Then either delete the original project or rename it to indicate it has been archived.
-    :::note
-    Make sure to [update any test schedules](/api-testing/schedule-test/) for tests in the archived project so that tests are not run automatically.
-    :::
+Suppose an organization admin deletes a team from Sauce Labs [Accounts > Team Management](/basics/acct-team-mgmt/managing-user-info/) before managing project access in API Testing. In that case, you see a banner at the top of the Projects page that notifies you some projects belong to a deleted team. Clicking on the **Update** brings you to the Settings, where you can assign a new access level to that project.
```

Comment:
Is it true that the access level defaults to Organization? The change that was made  is that it can now be updated. 
If so, maybe this would work?
```suggestion
If an organization admin deletes a team from Sauce Labs [Accounts > Team Management](/basics/acct-team-mgmt/managing-user-info/) before managing project access in API Testing, the access level defaults to Organization, because the access setting is required. However, a new access level can be assigned to the project. 

To update the project access:
1. Click **Update** in the banner message or navigate to the project settings General page.
2. In Project Details,  select the **Access** level for the project. If you are a team member you can select your team or organization-wide access. If you are an organization admin, you can select any team or organization-wide access. 
```

Reply:
@aliciakrebs No, there's no default value. Since the level can be changed, the project has no access level set at all, and the user needs to change the level to Organization or "Team".